mdbenchmark/plot.py

```diff
@@ -0,0 +1,271 @@
+    if ax is None:
+        ax = plt.gca()
+
+    p = ax.plot(df_sel, 'ns/day', '.-', data=df, ms='10', label=label)
+    color = p[0].get_color()
+    slope, intercept = calc_slope_intercept(
+        (df[df_sel].iloc[0], df['ns/day'].iloc[0]), (df[df_sel].iloc[1],
+                                                     df['ns/day'].iloc[1]))
+    # avoid a label and use values instead of pd.Series
+    ax.plot(
+        df[df_sel],
+        lin_func(df[df_sel].values, slope, intercept),
+        ls='--',
+        color=color,
+        alpha=.5)
+    return ax
+
+
+def plot_over_group(df, plot_cores, ax=None):
+    # plot all lines
+    df_sel = 'ncores' if plot_cores else 'nodes'
+
+    gb = df.groupby(['gpu', 'module', 'host'])
+    groupby = ['gpu', 'module', 'host']
+    for key, df in gb:
+        label = ' '.join(['{}={}'.format(n, v) for n, v in zip(groupby, key)])
+        plot_line(df=df, df_sel=df_sel, ax=ax, label=label)
+
+    # style axes
+    xlabel = 'cores' if plot_cores else 'nodes'
+    ax.set_xlabel('Number of {}'.format(xlabel))
+    ax.set_ylabel('Performance [ns/day]')
+    ax.legend()
+
+    return ax
+
+
+@cli.command()
+@click.option(
+    '--csv',
+    help='name of csv file',
+    multiple=True,
+    show_default=True)
+@click.option(
+    '--output-name',
+    '-o',
+    default=None,
+    help="name of output files",
+    show_default=True)
+@click.option(
+    '--output-type',
+    '-t',
+    help="file extension for plot outputs",
+    type=click.Choice(['png', 'pdf', 'svg', 'ps']),
+    show_default=True,
+    default='png')
+@click.option(
+    '--host-name',
+    '-h',
+    default=None,
+    multiple=True,
+    help="hostname",
+    show_default=True)
+@click.option(
+    '--module-name',
+    '-h',
+    default=None,
+    multiple=True,
+    help="module name or engine name (gromacs, namd)",
+    show_default=True)
+@click.option(
+    '--host-name',
+    '-h',
+    default=None,
+    multiple=True,
+    help="hostname",
+    show_default=True)
+@click.option(
+    '--gpu/--no-gpu',
+    help="plot data for GPU runs",
+    show_default=True,
+    default=False)
+@click.option(
+    '--cpu/--no-cpu',
+    help="plot data for CPU runs",
+    show_default=True,
+    default=True)
+@click.option(
+    '--plot-cores',
+    help="Plot performance per core instead of nodes",
+    show_default=True,
+    is_flag=True)
+def plot(csv, output_name, output_type, host_name, module_name, gpu, cpu, plot_cores):
+    """Generate plots from csv files"""
+
+    df = pd.DataFrame()
+
+    df_list = []
+
+    if len(csv) == 0:
+        console.error("No csv file specified. use --csv to specify a file.")
+
+    for c in csv:
+
+        tmp_df = pd.read_csv(c, index_col=0)
+        # append df_list
+        df_list.append(tmp_df)
+
+    df = pd.concat(df_list)
+
+    # Remove NaN values. These are missing ncores/performance data.
+    df = df.dropna()
+
+    if not gpu:
+        df = df[df['gpu'].isin([False])]
```

Comment:
`df = df[not df.gpu]`

Reply:
I think the official boolean indexing would be `df = df[~df.gpu]`, see `pandas` documentation: https://pandas.pydata.org/pandas-docs/stable/indexing.html#boolean-indexing